packages/example/next.config.js

```diff
@@ -3,9 +3,9 @@
  */
 const nextConfig = {
   env: {
-    infuraKey: '84842078b09946638c03157f83405213',
```

Comment:
Thanks for repo. Keys are leaked FYI.

Reply:
np - this is a public testing key from https://github.com/ethers-io/ethers.js/blob/33a029e457320a226c68a01f4cfa2d110125a8b8/packages/providers/src.ts/infura-provider.ts#L17

Reply:
Ok great! Didn't want you getting an unexpected bill... cheers!